### ISSUE TITLE
jpackage `create-image` has been renamed to `create-app-image`

### ISSUE THREAD

**Issue by Conorrr** [beryx/badass-jlink-plugin#53]
When I try to run the jpackage task against the latest ea release of jpackage(Build 49) I receive the following error:
```
Starting process 'command '/home/user/localjava/jdk-jpackage-13//bin/jpackage''. Working directory: /home/user/company/application Command: /home/user/localjava/jdk-jpackage-13//bin/jpackage create-image --output /home/user/company/application/build/jpackage --name em --module-path /home/user/company/application/build/jlinkbase/jlinkjars --module com.company/com.company.application.Module --runtime-image /home/user/company/application/build/image
Successfully started process 'command '/home/user/localjava/jdk-jpackage-13//bin/jpackage''
Error: Invalid Option: [create-image].
```

It seems that `create-image` has been changed to `create-app-image` see [jpackage docs](https://openjdk.java.net/jeps/343#Command-line-options)

**Comment by trejkaz** [comment]
I just pulled the current jpackage based on jdk-14 from the current download page here:
https://jdk.java.net/jpackage/

In this version, I get:

Error: Invalid Option: [create-app-image].

So it seems they either changed it again, or the version posted on that site isn't the latest one I can get my hands on.

Which one should I be using? A lot of guides on jpackage say to get the JDK13 version from the URL above, but the URL above has one based on JDK14.

**Comment by siordache** [comment]
There are several breaking changes in the CLI of the jdk-14 build 8 version of jpackage.

The next release of this plugin will support the new CLI. 

Until then, you can use the previous jpackage binaries:

[Linux/x64](https://download.java.net/java/early_access/jpackage/49/openjdk-13-jpackage+49_linux-x64_bin.tar.gz)
[macOS/x64](https://download.java.net/java/early_access/jpackage/49/openjdk-13-jpackage+49_osx-x64_bin.tar.gz)
[Windows/x64](https://download.java.net/java/early_access/jpackage/49/openjdk-13-jpackage+49_windows-x64_bin.zip)

**Comment by siordache** [comment]
@trejkaz  I just released 2.11.2, which supports the jdk-14 build 8 of jpackage.
